src/main/java/pl/javastart/task/Square.java

```diff
@@ -0,0 +1,34 @@
+package pl.javastart.task;
+
+import java.util.Scanner;
+
+public class Square {
+    private int[] array;
+    private int sumOfArray;
+
+    void run(Scanner scanner) {
+        System.out.println("Podaj rozmiar tablicy:");
+        createArray(scanner.nextInt());
+        System.out.printf("Wprowadź %d liczb:\n", array.length);
+        fillArray();
+        sumOfArray();
+    }
```

Comment:
Ten kod możesz spokojnie przenieść do metody main. Metoda run(), która pojawia się w niektórych zadaniach pomaga nam w przypadku testów automatycznych, ale tutaj nie jest to potrzebne